FTBFS on [mips, mips64el, mipsel]: cannot use devNumber (type uint32) as type uint64

[mips](https://buildd.debian.org/status/fetch.php?pkg=runc&arch=mips&ver=1.0.0%7Erc5%2Bdfsg1-2&stamp=1529155516&raw=0), [mips64el](https://buildd.debian.org/status/fetch.php?pkg=runc&arch=mips64el&ver=1.0.0%7Erc5%2Bdfsg1-2&stamp=1529160601&raw=0) and [mipsel](https://buildd.debian.org/status/fetch.php?pkg=runc&arch=mipsel&ver=1.0.0%7Erc5%2Bdfsg1-2&stamp=1529158764&raw=0) exhibit the following FTBFS:

~~~~
# github.com/opencontainers/runc/libcontainer/devices
src/github.com/opencontainers/runc/libcontainer/devices/devices.go:34:25: cannot use devNumber (type uint32) as type uint64 in argument to unix.Major
src/github.com/opencontainers/runc/libcontainer/devices/devices.go:54:32: cannot use devNumber (type uint32) as type uint64 in argument to unix.Minor
~~~~

https://buildd.debian.org/status/package.php?p=runc&suite=unstable


Wrapping `devNumber` as `uint64(devNumber)` should help.

Affected lines:

https://github.com/opencontainers/runc/blob/master/libcontainer/devices/devices.go#L34
https://github.com/opencontainers/runc/blob/master/libcontainer/devices/devices.go#L54

--- a/libcontainer/devices/devices.go
+++ b/libcontainer/devices/devices.go
@@ -30,8 +30,9 @@ func DeviceFromPath(path, permissions string) (*configs.Device, error) {
 	}
 
 	var (
-		devNumber = stat.Rdev
+		devNumber = uint64(stat.Rdev)
 		major     = unix.Major(devNumber)
+		minor     = unix.Minor(devNumber)
 	)
 	if major == 0 {
 		return nil, ErrNotADevice
@@ -51,7 +52,7 @@ func DeviceFromPath(path, permissions string) (*configs.Device, error) {
 		Type:        devType,
 		Path:        path,
 		Major:       int64(major),
-		Minor:       int64(unix.Minor(devNumber)),
+		Minor:       int64(minor),
 		Permissions: permissions,
 		FileMode:    os.FileMode(mode),
 		Uid:         stat.Uid,

